inventory/pkg/client/client.go

```diff
@@ -624,6 +673,16 @@ func (client *inventoryClient) register() error {
 	zlog.InfraSec().Info().Msgf("Registered inventory client with UUID: %s", resp.ClientUuid)
 	client.uuidMutex.Unlock()
 
+	// Start the heartbeat ticker to keep the client connection alive.
+	go func(clientUUID string) {
+		if err := client.heartbeat(clientUUID); err != nil {
+			// heartbeat failed, close the stream and connection.
+			zlog.InfraSec().Info().Msgf("heartbeat stopped for client UUID: %s", clientUUID)
```

Comment:
Since this is an error condition, we should include the error message rather than the Info message

Reply:
Indeed, fixed.